<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>monitoringWindow</class>
 <widget class="QWidget" name="monitoringWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1334</width>
    <height>830</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTableWidget" name="monitoringTable">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>1061</width>
     <height>771</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>1090</x>
     <y>10</y>
     <width>230</width>
     <height>102</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout2">
    <property name="fieldGrowthPolicy">
     <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
    </property>
    <item row="0" column="0">
     <widget class="QLabel" name="colorCode">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>100</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>6</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>30</number>
      </property>
      <item>
       <widget class="QLabel" name="yellowLabel">
        <property name="text">
         <string>UPS Satus</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="greenLabel">
        <property name="text">
         <string>Thermal Status</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>OVP Fast Satus</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="StartPush">
   <property name="geometry">
    <rect>
     <x>1120</x>
     <y>260</y>
     <width>121</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
  </widget>
  <widget class="QPushButton" name="StopPush">
   <property name="geometry">
    <rect>
     <x>1120</x>
     <y>300</y>
     <width>121</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ShutdownPush">
   <property name="geometry">
    <rect>
     <x>1120</x>
     <y>340</y>
     <width>121</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>EmergencyShutdown</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
